src/app/exams/exams-question.component.ts

```diff
@@ -64,4 +64,12 @@ export class ExamsQuestionComponent implements OnInit {
     return item.id;
   }
 
+  onChange(type) {
+    if (this.choices.length > 0) {
```

Comment:
This if statement is unnecessary.  The for loop will never be entered if length is 0.